src/test/jtx/impl/Env.cpp

```diff
@@ -124,11 +125,17 @@ Env::close(NetClock::time_point closeTime,
         app().getOPs().acceptLedger(consensusDelay);
     else
     {
-        rpc("ledger_accept");
+        auto resp = rpc("ledger_accept");
+        if (resp["result"]["status"] != std::string("success"))
+        {
+            std::cerr << "Env::close() failed: " << resp["result"]["status"] << std::endl;
+            res = false;
```

Comment:
I can't imagine any situation where `ledger_accept` would not return success, but I know that you encountered at least one instance; I think it's worth understanding what happened there, because other that not running in standalone mode, there's no way the operation can fail and the unit tests essentially "force" standalone mode.

I would definitely not use `std::cerr` to log from here; use `test.log << "whatever"` instead.

Reply:
Several failures of Env::close() appears on almost every local run or Travis run, you can see the Travis log for example: [https://travis-ci.com/github/ripple/rippled/jobs/320158724)](https://travis-ci.com/github/ripple/rippled/jobs/320158724), lines 713-721 and 773-775. I used std::cerr to see these messages on Travis log with --quiet option.

Reply:
Writing to std::cerr replaced by writing to journal.